.php_cs

```diff
@@ -11,7 +11,7 @@ EOF;
 Symfony\CS\Fixer\Contrib\HeaderCommentFixer::setHeader($header);
 
 return Symfony\CS\Config\Config::create()
-    ->level('symfony')
+    ->level(Symfony\CS\FixerInterface::SYMFONY_LEVEL)
```

Comment:
I'd be in favor of keeping it explicitly set. It eases understanding which level is set in case you don't know which one is the default. It also allows to straightforwardly change the constant (by knowing where to search for and saving some typing :p )